config/locales/client.en.yml

```diff
@@ -2221,8 +2221,8 @@ en:
         other: "There are <b>%{count}</b> replies."
       buttons:
         hide: "Hide summary"
-        generate: "Summarize with AI"
-        regenerate: "Regenerate summary"
+        generate: "Summarize"
```

Comment:
I'm wondering if it should be "Summary" instead :thinking:

Reply:
"SummArIze" perhaps lol